reporting_app/util.py

```diff
@@ -1,6 +1,9 @@
 import requests
 from flask import json
 from config import reporting_app_config as cfg, col_mappings
+import numpy
+import math
+import datetime
 
```

Comment:
It looks like  numpy, math, and datetime are not used here.
numpy also add a unnecessary dependency